scripts/generate_test_keys.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+echo "Generating aries-framework-go Test PKI"
+mkdir -p test/fixtures/keys/tls
```

Comment:
Suggest a more generic name than "key"/"keys", since I see x509 certificates here too.

Reply:
'pki' maybe?

Reply:
yes, pki seems okay (or possibly crypto).

Reply:
Aslo, if these are fixtures for the http transport testing then they should be underneath a testate folder (under transport/http)

Reply:
sure will rename to `crypto` and move folder under transport/http/.. as opposed to test/fixtures/...

Reply:
(note: under transport/http/testdata)

Reply:
sure.. this will be done in another pr though